Automattic/woocommerce-payments
# Declare HPOS incompatibility

#### Changes proposed in this Pull Request

Declare incompatibility of WooCommerce Payments with HPOS: pcShBQ-oY-p2

#### Testing instructions
- Get latest HPOS WooCommerce release from [here](https://github.com/woocommerce/woocommerce/releases/tag/feature-custom-order-table)
- Run the following command using wp cli 
```
wp eval 'do_action("before_woocommerce_init"); print_r(Automattic\WooCommerce\Utilities\FeaturesUtil::get_compatible_plugins_for_feature("custom_order_tables"));'
```
- Should get output similar to below, declaring WooCommerce Payments as an incompatible plugin
```
Array
(
    [compatible] => Array
        (
        )

    [incompatible] => Array
        (
            [0] => woocommerce-payments/woocommerce-payments.php
        )

)
```
- Backward Compatibility: On  7.0/ older versions of WooCommerce, the above command will not run, but the plugin should function without any existing functionality breaking.

-------------------

- [x] N/A - Covered with tests (or have a good reason not to test in description ☝️)
- [x] N/A - Tested on mobile (or does not apply)

**Post merge**



- [x] N/A - Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
